src/memo/memo.ml

```diff
@@ -145,9 +145,15 @@ module Id = Id.Make ()
 module Caches = struct
   let cleaners = ref []
```

Comment:
Do we really need two separate lists for cleaners and invalidators?
Since we seem to always be using them together, I think we should just fuse them together.

Reply:
Agreed. Initially I thought we might want to sometimes only invalidate all nodes without dropping the table contents, but now I'm not sure we'll ever want that. I'll change to having just one list.

Reply:
Done.